Xamarin.Forms.Core/Interactivity/TriggerBase.cs

```diff
@@ -122,8 +122,8 @@ public SealedList()
 
 			public void Add(T item)
 			{
-				if (IsReadOnly)
-					throw new InvalidOperationException("This list is ReadOnly");
+				//if (IsReadOnly)
```

Comment:
Why keep comments?

Reply:
because this is not right, needs to be fixed